daemon/k8s_watcher.go

```diff
@@ -982,6 +992,46 @@ func (d *Daemon) deleteK8sEndpointV1(ep *v1.Endpoints) {
 	endpointMetadataCache.delete(ep)
 }
 
+// missingK8sEndpointsV1 returns a map containing missing endpoints considered
+// missing from the k8sEndpoints of the loadbalancer.
+func (d *Daemon) missingK8sEndpointsV1(m versioned.Map) versioned.Map {
```

Comment:
It would be nice if the name of this function indicated that it's working on the loadbalancer endpoints, e.g., `missingLoadBalancerK8sEndpointsV1()`

Reply:
I can do it later on for all the `missing` functions.